### PR TITLE
fix/journey-book-50: separate css styles into an external style sheet

### LINKED ISSUE #50: [STYLE] Separate css styles into an external style sheet
**Is your feature request related to a problem? Please describe.**
Hi, My suggestion is to separate CSS in the `index.html` file into a CSS file called `style.css`. It will make CSS cleaner and more organized.

**Screenshot**

image

### PR DESCRIPTION
**Pull request by ivanetra**
# Description

Fixes #50: Separate CSS styles into an external style sheet. It makes the CSS cleaner and more organized.

# Checklist:

- [x] I have performed a self-review of my own code
- [x] My changes generate no new warnings

### CODE REVIEW

File: Data.json [hunk 1]
```diff
@@ -14,9 +14,14 @@
...
     "avatar": "https://github.com/vikasganiga05.png"
   },
-    {
+  {
     "name": "Shourjjo Majumder",
     "username": "shourgamer2",
     "avatar": "https://github.com/shourgamer2.png"
+  },
+  {
+    "name": "Ivan Eka Putra",
```
Comment (KendallDoesCoding) [inline Data.json:23]:
```suggestion
    "name": "Ivan Eka Putra",
```

Hi @ivanetra 

Thanks for your contribution, please keep the journey as a separate PR and fixing #50 as a separate PR.

Also, you haven't created a journey, you have only added the .json, please create a journey in the journeys directory.

the json helps the website link your journey properly over there, but is useless if it has nothing to link to :)

Thanks!

Comment (KendallDoesCoding) [reply Data.json:23]: I am deleting the data.json code which you have added please create a journey and add this json code again in a new PR, sorry for the inconvenience. 

Thanks

**Review by KendallDoesCoding — APPROVED** [review]

File: Data.json (hunk 1 above)
Comment (ivanetra) [reply Data.json]: Thank you @KendallDoesCoding !

**Review by vikasganiga05 — CHANGES_REQUESTED** [review]
Please Revert the changes from the `package-lock.json` file. Let us know if you need any help!

**Comment by KendallDoesCoding** [comment]
> Please Revert the changes from the `package-lock.json` file. Let us know if you need any help!

I reverted it in [dd62a82](https://github.com/collab-community/journey-book/pull/54/commits/dd62a82bdd376e1547ac4be3d16ed1c397c0b671)

**Review by KendallDoesCoding — APPROVED** [review]

**Comment by KendallDoesCoding** [comment]
Hey @ivanetra , thanks for your contribution on this, please create a journey for yourself :)

**Comment by ivanetra** [comment]
Anytime @KendallDoesCoding . Thank you!
